src/nft/components/NftMint.tsx

```diff
@@ -0,0 +1,64 @@
+import { background, border, cn, color } from '../../styles/theme';
+import { useIsMounted } from '../../useIsMounted';
+import { useTheme } from '../../useTheme';
+import { LifecycleType, type NftMintReact } from '../types';
+import { NftLifecycleProvider } from './NftLifecycleProvider';
+import { NftMintProvider } from './NftMintProvider';
+import { NftProvider } from './NftProvider';
+
+export function NftMint({
+  children,
+  className,
+  contractAddress,
+  tokenId = '1',
```

Comment:
are NFT `tokenIds` 0-indexed?

Reply:
seems like this is also assuming the NFT is going to be the first mint - is it a common developer workflow to provide the specific `tokenId` you're going to mint? I would assume that's handled by the intrinsic smart contract logic

Reply:
some are 0 indexed, not all, depends on the contract.  I've removed this default and will handle based on contract type in the data/mint hooks.